src/Checkbox.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import cx from 'classnames';
+import Flex from './Flex';
+import FlexItem from './FlexItem';
+
+/**
+ * @module Checkbox
+ */
+class Checkbox extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			checked: props.checked || false
+		};
+		this.onChange = this.onChange.bind(this);
+	}
+
+	onChange(e) {
+		this.setState({ checked: e.target.checked });
+	}
+
+	render() {
+		const {
+			checked, // eslint-disable-line no-unused-vars
+			children,
+			className,
+			id,
+			label,
+			name,
+			value,
+			...other
+		} = this.props;
+
+		const classNames = cx(
+			'minTouchHeight',
+			className
+		);
+
+		const elId = id || `${name}-${value}`;
+
+		return (
+			<Flex align='center'
+				className={classNames}
+				{...other}>
+				<FlexItem shrink>
+					<input	type='checkbox'
+						name={name}
+						value={value}
+						checked={this.state.checked}
+						id={elId}
+						onChange={this.onChange}
+					/>
+				</FlexItem>
+				<FlexItem>
+					<label className='label--minor' htmlFor={elId}>{label}</label>
```

Comment:
TL;DR: Checkbox labels shouldn't be bold

The `label--minor` class was something we were playing with to "un-bold" `<label>` for radio and checkbox inputs. We wanted to use the bold text for `<legend>` when we have a set of checkboxes that need to be labeled. 
E.g.: ``` <fieldset>
    <legend>Colors</legend>
    <input type="checkbox" id="red">
    <label for="red">Red</label>
    <input type="checkbox" id="blue">
    <label for="blue">Blue</label>
  </fieldset>```

The CSS behind `label--minor` didn't make it into this repo, it should probably go in [swarm-sasstools](https://github.com/meetup/swarm-sasstools)